src/servers/main.cc

```diff
@@ -213,9 +217,12 @@ std::vector<Option> options_{
      "controlled by 'repository-poll-secs'."},
     {OPTION_POLL_REPO_SECS, "repository-poll-secs",
      "Interval in seconds between each poll of the model repository to check "
-     "for changes. A value of zero indicates that the repository is checked "
-     "only a single time at startup. Valid only when "
+     "for changes. Valid only when "
      "--allow-poll-model-repository=true is specified."},
+    {OPTION_ALLOW_MODEL_CONTROL, "allow-model-control",
+     "Allow to load or to unload models explicitly using model control API. "
+     "If true the model repository will not be polled at startup. Ignored "
```

Comment:
"If true the models in the model repository will not be loaded at startup. Cannot be specified if --allow-pool-model-repository is true."